packages/site/src/main.css

```diff
@@ -91,6 +91,13 @@ section[role="tabpanel"] {
 
 }
 
+pre {
+  background-color: var(--color-surface--background);
+  border-radius: var(--radius-base);
+  border: var(--border-base) solid var(--color-border);
+  margin: 0 !important;
```

Comment:
Is there any way to do this without `important!`? It probably should be avoided if possible

Reply:
Probably 😅  I just need to figure out where the margin that was over-riding this property comes from and make a more specific selector

Reply:
Adjusted: https://github.com/GetJobber/atlantis/pull/2181/commits/1da2f9ef14ad5d7099d82df358a09607ea71cac2